### ISSUE TITLE
Issue committing match

### ISSUE THREAD

**Issue by jvens** [FIRST-Tech-Challenge/scorekeeper#451]
At an event this evening there was an issue with a match where the following error came up on the screen.  This event was using paper scoring.  @rebeccawhitaker was the scorekeeper at the event if you have further questions.

![image](https://user-images.githubusercontent.com/7074504/68911500-d5754d00-071a-11ea-9079-f5735c8ef436.png)

[lothal_2019_11_14_meet5.zip](https://github.com/FIRST-Tech-Challenge/scorekeeper/files/3849518/lothal_2019_11_14_meet5.zip)
[2019_11_14_05_25_23_PM_err.log](https://github.com/FIRST-Tech-Challenge/scorekeeper/files/3849520/2019_11_14_05_25_23_PM_err.log)

**Comment by adwiii** [comment]
@jvens @rebeccawhitaker the database has the same final score as what is shown in the picture, blue 29, red 2, which I assume is correct. It also shows that the score was committed 3 times. Did the error occur more than once in those 3 attempts?

**Comment by rebeccawhitaker** [comment]
Yes, the final score was blue 29, red 2. Yes, this error occurred several times and I hit "ok" and clicked "save" and "commit score" until this box did not appear. Hope this helps!

**Comment by jvens** [comment]
Close as cannot replicate.  This issue has not occurred again.
